examples/hypershift_destroy.yaml

```diff
@@ -0,0 +1,13 @@
+# hypershift installation
+hypershift_destroy: false
+mgmt_cluster_kubeconfig: "{{ ansible_env.HOME }}/.kube/config"
+hosted_cluster_name: "rdr-hypershift-cluster"
+hypershift_dir: "/root/hypershift"
+hypershift_git_repo: "https://github.com/openshift/hypershift"
+hypershift_git_branch: "main"
+ibmcloud_resource_group: ""
+powervs_region: ""
+powervs_zone: ""
+vpc_region: ""
+base_domain: ""
```

Comment:
Can we show some example values here? These are required so cannot expect the user to pass an empty string.